src/sql/pg/create.sql

```diff
@@ -1,69 +1,141 @@
+  "NAME" CHARACTER VARYING(255) NOT NULL,
+  "BALANCE" DECIMAL(32,16) NOT NULL CHECK
+    ("BALANCE" >= "MINIMUM_ALLOWED_BALANCE"),
+  "CONNECTOR" CHARACTER VARYING(1024) NULL,
+  "PASSWORD_HASH" CHARACTER VARYING(1024) NULL,
+  "PUBLIC_KEY" CHARACTER VARYING(4000) NULL,
+  "IS_ADMIN" SMALLINT NOT NULL,
+  "IS_DISABLED" SMALLINT NOT NULL,
+  "FINGERPRINT" CHARACTER VARYING(255),
+  "MINIMUM_ALLOWED_BALANCE" DECIMAL(32,16) default 0
+);
+
+CREATE INDEX "L_XPK_ACCOUNTS" ON "L_ACCOUNTS"
+  ("ACCOUNT_ID" ASC);
+ALTER TABLE "L_ACCOUNTS" ADD CONSTRAINT "L_PK_ACCOUNTS" PRIMARY KEY
+  ("ACCOUNT_ID");
+CREATE UNIQUE INDEX "L_XAK_ACCOUNTS" ON "L_ACCOUNTS"
+  ("NAME" ASC);
+ALTER TABLE "L_ACCOUNTS" ADD CONSTRAINT "L_AK_ACCOUNTS" UNIQUE
+  ("NAME");
+CREATE INDEX "L_XIE_FINGERPRINT" ON "L_ACCOUNTS"
+  ("FINGERPRINT");
+
+
+CREATE TABLE IF NOT EXISTS "L_TRANSFERS" (
+  "TRANSFER_ID" CHARACTER VARYING(36) NOT NULL,
+  "LEDGER" CHARACTER VARYING(1024),
+  "DEBITS" CHARACTER VARYING(4000),
+  "CREDITS" CHARACTER VARYING(4000),
+  "ADDITIONAL_INFO" CHARACTER VARYING(4000),
+  "STATE" CHARACTER VARYING(20),
+  "REJECTION_REASON" CHARACTER VARYING(10),
+  "EXECUTION_CONDITION" CHARACTER VARYING(4000),
+  "CANCELLATION_CONDITION" CHARACTER VARYING(4000),
+  "EXPIRES_AT" TIMESTAMP WITH TIME ZONE NULL,
+  "PROPOSED_AT" TIMESTAMP WITH TIME ZONE NULL,
+  "PREPARED_AT" TIMESTAMP WITH TIME ZONE NULL,
+  "EXECUTED_AT" TIMESTAMP WITH TIME ZONE NULL,
+  "REJECTED_AT" TIMESTAMP WITH TIME ZONE NULL
+);
+
+CREATE INDEX "L_XPK_TRANSFERS" ON "L_TRANSFERS"
+  ("TRANSFER_ID" ASC);
+ALTER TABLE "L_TRANSFERS" ADD CONSTRAINT "L_PK_TRANSFERS" PRIMARY KEY
+  ("TRANSFER_ID");
+-- CREATE INDEX "L_XAK_TRANSFERS" ON "L_TRANSFERS"
+--   ("TRANSFER_UUID" ASC);
+-- ALTER TABLE "L_TRANSFERS" ADD CONSTRAINT "L_AK_TRANSFERS" UNIQUE
+--   ("TRANSFER_UUID");
+CREATE INDEX "L_XIF_TRANSFERS_STATE" ON "L_TRANSFERS"
+  ("STATE" ASC);
+CREATE INDEX "L_XIF_TRANSFERS_REASON" ON "L_TRANSFERS"
+  ("REJECTION_REASON" ASC);
+
+
+CREATE TABLE IF NOT EXISTS "L_SUBSCRIPTIONS" (
+  "SUBSCRIPTION_ID" CHARACTER VARYING(36) NOT NULL,
+  "OWNER" CHARACTER VARYING(255) NOT NULL,
+  "EVENT" CHARACTER VARYING(255) NOT NULL,
+  "SUBJECT" CHARACTER VARYING(1024) NOT NULL,
+  "TARGET" CHARACTER VARYING(1024) NOT NULL,
+  "IS_DELETED" BOOLEAN NOT NULL DEFAULT FALSE
+);
+
+CREATE INDEX "L_XPK_SUBSCRIPTIONS" ON "L_SUBSCRIPTIONS"
+  ("SUBSCRIPTION_ID" ASC);
+ALTER TABLE "L_SUBSCRIPTIONS" ADD CONSTRAINT "L_PK_SUBSCRIPTIONS" PRIMARY KEY
+  ("SUBSCRIPTION_ID");
+-- CREATE UNIQUE INDEX "L_XAK_SUBSCRIPTIONS" ON "L_SUBSCRIPTIONS"
+--   ("SUBSCRIPTION_UUID" ASC);
+-- ALTER TABLE "L_SUBSCRIPTIONS" ADD CONSTRAINT "L_XAK_SUBSCRIPTIONS" UNIQUE
+--   ("SUBSCRIPTION_UUID");
+CREATE INDEX "L_XIF_SUBSCRIPTIONS_OWNER" ON "L_SUBSCRIPTIONS"
+  ("OWNER" ASC);
+CREATE INDEX "L_XIF_SUBSCRIPTIONS_SUBJECT" ON "L_SUBSCRIPTIONS"
+  ("SUBJECT" ASC);
+CREATE INDEX "L_XIE_SUBSCRIPTIONS_DELETED" ON "L_SUBSCRIPTIONS"
+  ("IS_DELETED" ASC);
```

Comment:
You have join index on `is_deleted` and the `id` for sqlite, but not here. What's the difference?